src/components/SubRequirement.vue

```diff
@@ -32,24 +32,20 @@
         <span>{{subReq.minCountFulfilled}}/{{subReq.minCountRequired}} {{ subReq.fulfilledBy }}</span>
       </p>
     </div>
-  </div>
   <div v-if="displayDescription" :class="[{'completed-ptext': this.isCompleted}, 'description']">
     {{ subReq.requirement.description }} <a class="more"
     :style="{ 'color': `#${color}` }"
     :href="subReq.requirement.source" target="_blank">
     <strong>Learn More</strong></a>
     <div v-if="subReq.requirement.fulfilledBy === 'toggleable'">
       <div class="toggleable-requirements-select-wrapper">
-        <div
-          class="toggleable-requirements-select toggleable-requirements-input"
-          v-click-outside="closeMenuIfOpen"
-        >
           <div
-            class="toggleable-requirements-dropdown-placeholder toggleable-requirements-dropdown-wrapper"
-            @click="showFulfillmentOptionsDropdown = !showFulfillmentOptionsDropdown"
+            class="toggleable-requirements-select toggleable-requirements-input"
+            v-click-outside="closeMenuIfOpen"
```

Comment:
Why is this being changed?

Reply:
This was from resolving a merge conflict when I was merging master into the branch. I already had toggleable requirements merged in at this point so the latest master had some additional changes. The div with class `"toggleable-requirements-dropdown-placeholder toggleable-requirements-dropdown-wrapper"` isn't removed but can be found below.